data/simple-icons.json

```
@@ -6786,6 +6786,13 @@
 		"hex": "000000",
 		"source": "https://graphite.dev"
 	},
+	{
+		"title": "Graphite",
+		"slug": "graphite_editor",
+		"hex": "473A3A",
+		"source": "https://graphite.art/logo",
+		"guidelines": "https://graphite.art/logo"
```

Quoting the terms of service, I think that we can add the license as well:

> The Graphite logo is made available for community use, with some limitations. While the software is free and open source, the brand identity is more restrictive. Please be respectful of the Graphite brand by reviewing this usage policy.
>
> Be aware that the logo is not covered under the [Apache License 2.0](https://graphite.art/license#source-code) of Graphite's source code. Its usage within Graphite software distributions is covered by the [Graphite Branding License](https://graphite.art/license#branding).

```suggestion
		"guidelines": "https://graphite.art/logo",
		"license": {
		    "type": "custom",
		    "url": "https://graphite.art/license/#branding"
		}
```